src/test/resources/pages/instructorSearchPageSearchFeedbackResponseComments.html

```diff
@@ -216,4 +216,4 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
I'm guessing they have more than one newline between blocks and/or trailing spaces.